diagnostic.md

```diff
@@ -2,7 +2,8 @@
 
 Record your responses inside the fenced code blocks below each question.
 
-1.  Give an example of a visual hierarchy that could be modeled with components.
+1.  Give an example of a visual hierarchy that could be modeled with components
+(draw a wireframe).
```

Comment:
How can one draw a wireframe on the diagnostic? Please provide instructions to clarify that this is a problem solving technique or is required for part of the answer. If neither, remove this instruction. If required as part of the answer, indicate that the file should be hosted and linked here.